packages/http-client-csharp/generator/TestProjects/Local/Sample-TypeSpec/src/Generated/Models/Thing.cs

```diff
@@ -130,22 +130,22 @@ internal Thing(BinaryData requiredUnion, ThingRequiredLiteralString requiredLite
         public string OptionalNullableString { get; set; }
 
         /// <summary> required literal int. </summary>
-        public ThingRequiredLiteralInt RequiredLiteralInt { get; } = 123;
+        public int RequiredLiteralInt { get; } = 123;
 
         /// <summary> required literal float. </summary>
-        public ThingRequiredLiteralFloat RequiredLiteralFloat { get; } = 1.23F;
+        public float RequiredLiteralFloat { get; } = 1.23F;
 
         /// <summary> required literal bool. </summary>
         public bool RequiredLiteralBool { get; } = false;
 
         /// <summary> optional literal string. </summary>
-        public ThingOptionalLiteralString? OptionalLiteralString { get; set; }
+        public string OptionalLiteralString { get; set; }
 
         /// <summary> optional literal int. </summary>
-        public ThingOptionalLiteralInt? OptionalLiteralInt { get; set; }
+        public int? OptionalLiteralInt { get; set; }
```

Comment:
so maybe we need to have an issue about why this xml doc is not showing the possible accepted values since this is an optional literal, only one value other than null would be acceptable.
We have a mechanism that if the type is literal, we would write something in the xml doc, but somehow that is not working here.

Reply:
Filed https://github.com/microsoft/typespec/issues/7410